src/components/RadioButtonGroup/readme.md

```diff
@@ -268,3 +268,187 @@ class ErrorRadioButtonGroup extends React.Component {
+        email: 'leandro@gmail.com',
+        linkedin: '#',
+        github: '#',
+        isActive: false,
+    },
+    {
+        avatar: 'images/user/avatar-5.svg',
+        name: 'Jose Torres',
+        email: 'jose@gmail.com',
+        linkedin: '#',
+        github: '#',
+        isActive: false,
+    },
+    {
+        avatar: 'images/user/avatar-6.svg',
+        name: 'Reinier',
+        email: 'reinier@gmail.com',
+        linkedin: '#',
+        github: '#',
+        isActive: false,
+    },
+    {
+        avatar: 'images/user/avatar-7.svg',
+        name: 'Sara',
+        email: 'sara@gmail.com',
+        linkedin: '#',
+        github: '#',
+        isActive: false,
+    },
+    {
+        avatar: 'images/user/avatar-8.svg',
+        name: 'Tahimi',
+        email: 'tahimi@gmail.com',
+        linkedin: '#',
+        github: '#',
+        isActive: false,
+    }
+];
+
+const Title = styled.h1.attrs(props => props.theme.rainbow)`
+    font-family: 'Lato';
+    font-size: 36px;
+    text-align: center;
+    color: ${props => props.palette.text.main};
+`;
+
+const StyledAvatar = styled(Avatar)`
+    width: 100px;
+    height: 100px;
+    box-shadow: 0 2px 4px 0 rgba(6, 28, 63, 0.3);
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+`;
+
+const NameLabel = styled.label.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 16px;
+    font-weight: 900;
+    text-align: center;
+    color: ${props => props.palette.text.main};
+    display: block;
+`;
+
+const EmailLabel = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 12px;
+    font-weight: 300;
+    text-align: center;
+    color: ${props => props.palette.text.label};
+    display: block;
+`;
+
+const ConctactContainer = styled.div.attrs(props => props.theme.rainbow)`
+    width: max-content;
+`;
+
+const Contact = (props) => {
+    const {avatar, name, email, linkedin, github} = props;
+    return (
+        <ConctactContainer className="rainbow-m-around_medium">
+            <StyledAvatar
+                src={ avatar }
+                assistiveText={ name }
+                title={ name }
+            />
+            <NameLabel className="rainbow-m-top_small">
+                { name }
+            </NameLabel>
+            <EmailLabel className="rainbow-m-top_x-small">
+                { email }
+            </EmailLabel>
+            <div className="rainbow-flex rainbow-justify_space-around rainbow-m-top_small">
+                <a href={linkedin}>
+                    <GitHubIcon />
+                </a>
+                <a href={github}>
+                    <LinkedinIcon />
+                </a>
+            </div>
+        </ConctactContainer>
+    );
+};
+
+const Contacts = ({data}) => {
+    const [showActive, setShowActive] = useState('active');
+
+    const options = [
+        { value: 'active', label: 'Actives' },
+        { value: 'all', label: 'All' },
+    ];
```

Comment:
since options are fixed and does not depends on props or state then put it outside the component because this will recreate a new array in each rerender